frontend/src/components/account.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import ReactDOM from 'react-dom/client';
+import { useState} from "react"; 
+import axios from 'axios';
+import { fchmod } from 'fs';
+import './signup.css';
+import { Link } from 'react-router-dom';
+
+export const Account = () => {
+
+
+    const [warning, setWarning] = useState<string>('');
+    const [changePassword, setChangePassword] = useState<string>('');
+
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+
+        e.preventDefault(); 
+        await axios.post('http://localhost:8000/account/profile', {
+            changePassword
+        });
```

Comment:
are these routes already defined and working or is this more a placeholder for now?